libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloClient.kt

```diff
@@ -243,6 +267,29 @@ private constructor(
       this.useV3ExceptionHandling = useV3ExceptionHandling
     }
 
+    private var ignorePartialData: Boolean? = null
```

Comment:
Can we make this on `MutableExecutionContext`? I'd like to be able to override this per-query